packages/compass-shell/package.json

```diff
@@ -56,9 +56,11 @@
     "@leafygreen-ui/icon-button": "^5.0.2",
     "@mongosh/browser-repl": "^0.5.2",
     "@mongosh/browser-runtime-electron": "^0.5.2",
+    "@mongosh/service-provider-core": "^0.5.2",
     "@mongosh/service-provider-server": "^0.5.2",
```

Comment:
I think you’ll need to rebase these :)

Reply:
Oh yeah, for sure! I was actually thinking that now that we have figured out all the pieces of the implementation, it might be easier to start from current master and cherry-pick/copy pieces of this PR into smaller, logical parts that would be easier to review instead of a massive PR like that